### PR TITLE
[BPK-2596] Fix Tappable Label protocol naming

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [-] `UNRELEASED.md`
+ [x] `README.md`
+ [-] Tests
+ [-] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### Still todo:
 - [x] Fix issue where text is cut off

### CODE REVIEW

File: Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabelDelegate.h [hunk 1]
```diff
@@ -25,6 +25,8 @@ NS_ASSUME_NONNULL_BEGIN
 @optional
 - (void)attributedLabel:(BPKTappableLinkLabel *)label didSelectLinkWithURL:(NSURL *)url;
 - (void)attributedLabel:(BPKTappableLinkLabel *)label didSelectLinkWithTransitInformation:(NSDictionary *)components;
```
Comment (k0nserv) [inline Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabelDelegate.h]: Can we add a comment about these two being deprecated?

File: Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabel.m [hunk 2]
```diff
@@ -261,12 +261,20 @@ - (void)attributedLabel:(BPKTappableLinkLabel *)label didSelectLinkWithURL:(NSUR
     if ([self.delegate respondsToSelector:@selector(attributedLabel:didSelectLinkWithURL:)]) {
         [self.delegate attributedLabel:label didSelectLinkWithURL:url];
     }
+
+    if ([self.delegate respondsToSelector:@selector(tappableLabel:didSelectLinkWithURL:)]) {
```
Comment (k0nserv) [inline Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabel.m]: I feel like we should only call one of these with a pereference for `tappableLabel:didSelectLinkWithURL:`

Comment (georgegillams) [reply Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabel.m]: Like this?
```
if (new_one_is_supported) {
  do new_one
} else if (old_one_is_supported) {
  do old_one
}
```

Comment (k0nserv) [reply Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabel.m]: Yeah that's what I am thinking

**Pull request merged**